### PR TITLE
feat:added click function dispalyed PO in dashboard

### PR DESCRIPTION
**Pull request by mllrdev-jerry-bayoneta**

### CODE REVIEW

File: src/shared/components/OrderList.vue [hunk 1]
```diff
@@ -36,7 +36,7 @@ export default defineComponent({
   },
   setup(props) {
     const purchaseOrderSequences = computed(() => {
```
Comment (mllrdev-carlo-medel) [inline src/shared/components/OrderList.vue]:
```suggestion
    const purchaseOrders = computed(() => {
```

File: src/shared/components/OrderList.vue [hunk 2]
```diff
@@ -49,6 +49,15 @@ export default defineComponent({
...
     })
     return { purchaseOrderSequences }
+  },
+  methods: {
+    handleOrderRowClick(id: string) {
+      const clickedOrder = this.purchaseOrderSequences.find((purchaseOrderSequences) => {
+        return purchaseOrderSequences.id === id;
+      });
+      console.log(clickedOrder);
+    }
```
Comment (mllrdev-carlo-medel) [inline src/shared/components/OrderList.vue]:
```suggestion
```
remove

File: src/shared/components/OrderList.vue (hunk 1 above)
Comment (mllrdev-jerry-bayoneta) [reply src/shared/components/OrderList.vue]: done

File: src/shared/components/OrderList.vue (hunk 2 above)
Comment (mllrdev-jerry-bayoneta) [reply src/shared/components/OrderList.vue]: done

**Review by mllrdev-carlo-medel — APPROVED** [review]

**Pull request merged**